test/run-chrome-smoketests.sh

```diff
@@ -0,0 +1,13 @@
+echo "\nTest webextension-polyfill from an extension running on chrome"
+echo "==============================================="
+
+export TEST_EXTENSIONS_PATH=/tmp/browser-polyfill-chrome-smoketests
+
+## Exits immediately if the directory already exists (which can only happen in a local
+## development environment, while this test will usually run on travis).
+mkdir $TEST_EXTENSIONS_PATH || exit 1
```

Comment:
I think it would be convenient for local testing if this script could be run repeatedly.

You could just remove the dir first which is probably pretty safe since it's a tmp dir:

```
rm -fr $TEST_EXTENSIONS_PATH
mkdir $TEST_EXTENSIONS_PATH || exit 1
```

Or if you wanted to be a bit paranoid you could check for a marker file to know if it's safe to delete:

```
MARKER_FILE=$TEST_EXTENSIONS_PATH/.created-for-run-chrome-smoketests
# Check if the marker file exists and then remove the directory
if [ ... $MARKER_FILE ]; then
  rm -fr $TEST_EXTENSIONS_PATH
fi
mkdir $TEST_EXTENSIONS_PATH || exit 1
touch $MARKER_FILE
```

And if you wanted to be *super* paranoid you could manually prefix `/tmp` for dangerous operations:

```
SUBDIR=browser-polyfill-chrome-smoketests
export TEST_EXTENSIONS_PATH=/tmp/$SUBDIR

MARKER_FILE=/tmp/$SUBDIR/.created-for-run-chrome-smoketests
# Check if the marker file exists and then remove the directory
if [ ... $MARKER_FILE ]; then
  rm -fr /tmp/$SUBDIR
fi
mkdir $TEST_EXTENSIONS_PATH || exit 1
touch $MARKER_FILE
```

Reply:
:+1: I like it

I've included the "moderately paranoid" version in the updated version of this patches :-)